src/index.tsx

```diff
@@ -1,19 +1,19 @@
-import * as serviceWorker from "./serviceWorker";
-import { Provider } from "react-redux";
-import ReactDOM from "react-dom";
-import * as React from "react";
+import * as serviceWorker from './serviceWorker';
```

Comment:
This isnt something that needs to be changed, but we should add a `.prettierrc` file so all of our autoformatters are identical

Reply:
Agreed, I think that was from me anyway 😅